### DISCUSSION TITLE
Diary-13-京爷吉祥！ | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#324] [Announcements]
# Diary-13-京爷吉祥！ | Zi-Zi's Journey

2024.03.25-2024.04.07

https://gz-metal-cell.github.io/2024/04/06/Diary-13-%E4%BA%AC%E7%88%B7%E5%90%89%E7%A5%A5%EF%BC%81/

**Comment by promefire** [comment]
Metal哥教我摄影艺术😭

**Reply by GZ-Metal-Cell** [reply]
😭数媒人虽然炼丹不如计科网工😭但是还是有自己的优势的😭

**Comment by promefire** [comment]
这个萌国ICP备案是个什么东东😭😭😭😭😭

**Reply by GZ-Metal-Cell** [reply]
😭走的时候看到 pro 神也在搞这玩意了!😭

**Reply by promefire** [reply]
😭俺也是萌国一份子了😭

**Reply by GZ-Metal-Cell** [reply]
😭pro 神仅需 45 分钟就从不知道萌国变成了萌国一份子😭他的不务正业能力太强了😭导师真是捡到宝了😭

**Reply by GZ-Metal-Cell** [reply]
😭[Zi-Zi's Journey](https://icp.gov.moe/?keyword=20241647) 是一首优美的[小提琴曲](https://music.163.com/#/song?id=20090401)，[Promefire 's Journey](https://icp.gov.moe/?keyword=20240622) 是个什么玩意啊😭东施效颦的 pro 哥😭

**Comment by promefire** [comment]
😭其实这是一首尚未发布的钢琴曲😭



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年4月13日(周六) 上午10:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [GZ-Metal-Cell/GZ-Metal-Cell.github.io] Diary-13-京爷吉祥！ | Zi-Zi's Journey (Discussion #324)




 
😭Zi-Zi's Journey 是一首优美的小提琴曲，Promefire 's Journey 是个什么玩意啊😭东施效颦的 pro 哥😭

**Comment by echs-top** [comment]
看起来是某两位学长捏，虽然和我明显学的不一样

**Reply by GZ-Metal-Cell** [reply]
本科是数字媒体技术专业🫣

**Comment by echs-top** [comment]
医～
